src/Mod/TechDraw/App/DrawViewDimension.cpp

```diff
@@ -686,8 +685,41 @@ std::string  DrawViewDimension::getFormatedValue(int partial)
     }
 
     return result;
+
 }
 
+QStringList DrawViewDimension::getFormattedToleranceValues(int partial)
+{
+    QString underFormatSpec = QString::fromUtf8(FormatSpecUnderTolerance.getStrValue().data());
+    QString overFormatSpec = QString::fromUtf8(FormatSpecOverTolerance.getStrValue().data());
+    QStringList tolerances;
+    QString underTolerance, overTolerance;
+
+    if (ArbitraryTolerances.getValue()) {
+        underTolerance = underFormatSpec;
+        overTolerance = overFormatSpec;
+    } else {
+        underTolerance = QString::fromUtf8(formatValue(UnderTolerance.getValue(), underFormatSpec, partial).c_str());
+        overTolerance = QString::fromUtf8(formatValue(OverTolerance.getValue(), overFormatSpec, partial).c_str());
+    }
+
+    tolerances << underTolerance << overTolerance;
+
+    return tolerances;
+}
+
+
+std::string DrawViewDimension::getFormattedDimensionValue(int partial)
+{
+//    Base::Console().Message("DVD::getFormattedValue(%d)\n", partial);

```

Comment:
These debugging printouts are actually not made by me, but they are leftovers by the previous developer, WandererFan, or even someone before him.  I have been a bit wary about touching them, as I am not a real FC developer, I mainly do this because WandererFan is absent.  This comment printout I just moved along when refactoring the code, but maybe I should take the courage the remove it..!